README.md

```diff
@@ -2,39 +2,52 @@
 
 [![PyPI Release](https://github.com/scailable/sclblpy/workflows/PyPI%20Release/badge.svg)](https://pypi.org/project/sclblpy/)
 
-`sclblpy` is the core python package provided by Scailable to convert models fit in python to WebAssembly and
-open them up as a REST endpoint.
 
-Currently the package supports the upload of fitted `sklearn` model objects, and it supports uploading `.onnx` models
-(by specifying a path to the `.onnx` file). The package can also be used to manage assignments of models to 
-registered devices. 
+Changelog: the package is currently undergoing a major rework. We try to keep the below up to date, but some features might be stubs. 
+Most relevant is the removal of support for Scikit learn in favor of focussing fully on ONNX.
+1. Removal of support for additional packages - most importantly in the 'upload()' function which now will throw an error if `model_type` isn't onnx
+2. the `run()` function is going to be replaced by a function that allows the user to test their local setup; and as such is a stub
+
+
+`sclblpy` is the core python package provided by Scailable for interacting with our API. The scope of this package is (roughly):
+1. upload an `.onnx` model to the admin console
+2. Assign and deploy the uploaded model to a device that has previously been installed with the Scailable runtime and registered
+3. (upcoming) test device deployment
+
+
+
+
+Also, there will probably be breaking changes in the API in one of the upcoming releases.
```

Comment:
"Going forward, there may be breaking changes in the API. Still, breaking changes will be avoided where possible. "